modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -10,6 +10,9 @@ import 'rxjs/add/operator/map';
 import * as models                                           from '../model/models';
 import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
 import { Configuration }                                     from '../configuration';
+{{#withInterfaces}}
+import { I{{classname}} }                                    from './{{classname}}Interface';
```

Comment:
If we look at the [coding guidlines from typescript](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines) the I prefix should be avoided.
If not to much work it would be beter to rename the api class that implements this interface with something like default.